backend/backend/settings/production.py

```diff
@@ -25,3 +25,13 @@
 # AZURE_ACCOUNT_KEY = os.environ['AZURE_STORAGE_ACCOUNT_KEY']
 # AZURE_CONTAINER = os.environ['AZURE_STORAGE_ACCOUNT_CONTAINER']
 # AZURE_LOCATION = os.environ['AZURE_STORAGE_ACCOUNT_LOCATION']
+
+# Used for serving local user-uploaded files
+# I.g., "http://127.0.0.1:8000/files/{file_path}"
+MEDIA_URL = 'files/'
+
+# Used for storing local user-uploaded files
+MEDIA_ROOT = os.path.join(BASE_DIR, MEDIA_URL)
+
+
+ALLOWED_HOSTS = os.environ['ALLOWED_HOSTS'].split(' ')
```

Comment:
Why is `ALLOWED_HOSTS` space-delimited in production, overriding the comma-delimited default in the base settings?

Reply:
Oops, no reason, I didn't even look to see if it was already defined in base.py.